### ISSUE TITLE
Racer and react

### ISSUE THREAD

**Issue by droganov** [derbyparty/derby-faq#47]
Hello.
I’d like to raise react.js question. What are the reasons to support derby these days? Wouldn’t it better to create racer + react + express setup instead of derby? As a result you would cut down the development costs (another year passed and derby still is .6 alpha) and get an access to tons of react components.

**Comment by droganov** [comment]
Hm zero feedback. What is happening around derby now?
I've [made](https://github.com/droganov/proton-starter) the setup with react, generally it works. I'd be appreciated for any useful feedback.

**Comment by vmakhaev** [comment]
Yes, I like React's View more than Derby's. And connecting React with Racer took me about 150 LOC through high order component. Though I like Racer's api, with React you don't need most of it, so it maybe does make sense to write another ShareJS wrapper for React.
Actually I'm writing one with Racer's api, but on CRDT (Offline support) instead of OT. Should be announced soon.

**Comment by droganov** [comment]
Is there any examples on the web to look at?
I like racer, it's OK for now. But it would be cool to make GraphQL — ShareJS adapter I guess.

**Comment by vmakhaev** [comment]
Not at the moment.

GraphQL and ShareJS very different conceptions.

**Comment by droganov** [comment]
Yes, that's tricky.

**Comment by droganov** [comment]
Technically we could link graphql server to share.js, but as I know they didn't publish realtime specs

**Comment by droganov** [comment]
Vladimir, are you working on wrapper for Swarm.js?
